var/spack/repos/builtin/packages/lcms/package.py

```diff
@@ -35,6 +37,27 @@ def url_for_version(self, version):
     depends_on("libtiff")
     depends_on("zlib-api")
 
+    build_system("autotools", "msbuild")
+
     @property
     def libs(self):
         return find_libraries("liblcms2", root=self.prefix, recursive=True)
+
+
+class MSBuildBuilder(spack.build_systems.msbuild.MSBuildBuilder):
+    @property
+    def build_directory(self):
+        return (
+            pathlib.Path(self.pkg.stage.source_path)
+            / "Projects"
+            / f"VC{self.pkg.compiler.visual_studio_version}"
+        )
+
+    def setup_build_environment(self, env):
+        jpeg_include = self.spec["jpeg"].prefix.include
+        tiff_include = self.spec["libtiff"].prefix.include
+        zlib_include = self.spec["zlib-api"].prefix.include
```

Comment:
Certainly less verbose/manual, but I wanted to be explicit/guard against anything else leaking its way into this. If you don't see that as a significant concern I'll refactor.

Reply:
I noticed that this was explicitly adding the `include/` dir for each dependency that this package `depends_on`, so I assumed that if another dependency was added, that you would want to account for it as well. If the build for `lcms` indicates that it specifically needs these packages to be included, then I think it's better the way you did it. If this is primarily to deal with the fact that the build on Windows doesn't inject dependencies into a compiler wrapper, and we need to communicate this info to a low-level build-system (e.g. vs. CMake, where CMAKE_PREFIX_PATH will help CMake locate all the Spack deps regardless of the wrapper), then I think the `.dependencies` loop would be more straightforward.

Reply:
The build system is MSBuild, so it relies on and expects the user to leverage standard MSVC mechanisms to provide the build various requirements required to complete the build. The build docs are relatively light (non existent) but this would be the "standard convention" for adding includes to an MSBuild run. That said, the compiler wrapper would also serve this purpose. 

I'll refactor to the suggested approach.